lib/commands/help.js

```diff
@@ -34,6 +44,35 @@ function activate(eName, message){
     return;
   }
 
-  return message.respond(`My commands are as follows:
-${JSON.stringify(help, null, 4)}`);
+  //handle general help
+  if(!msg[1]) {
+    let resp = 'Here are my commands:\n';
+    resp+= '```';
+    help.forEach((h)=>{
+      resp += `${_.padEnd(h.cmd, 12)}   ${h.description}\n`
+    });
+    resp+='```';
+    return message.respond(resp);
+  }
+
+  //handle help for a given command
+  let searchCmd = msg[1];
+  let resp = '```\n';
+  for(let i = 0; i < help.length; i++){
+    let cmd = help[i];
+    if(cmd.cmd === searchCmd || cmd.aliases.includes(searchCmd)){
```

Comment:
Not a big deal but once you wind up with something like `cmd.cmd` you likely need to come up with better named variables. `current.cmd` for instance. Or perhaps the problem is inherent in having a command with a variable of cmd within it. A command object could have keys of aliases, description, usage, and **keyword** or **main** etc.

Something to think about. But generally this is fine, not blocking my approval.